assets/base.css

```diff
@@ -2285,6 +2306,12 @@ input[type='checkbox'] {
       'navigation navigation navigation';
   }
 
+  .header--top-center.drawer-menu {
+    grid-template-areas:
+      '. left-icon heading icons';
```

Comment:
I don't think so because the `.` is a column itself in which the hamburger menu icon is. And only the search icon is associated to the `grid-area: left-icon;`
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/15-41-lfdtx-9l0xc.png)
</details>